<launch> 
  <rosparam file="$(find imu_gnss_fusion)/config/param_dual.yaml" command="load"/>
  <param name="pub_tf"           type="bool" value="true" />
  <param name="frame_id"         type="string" value="map_gnss" />
  <param name="child_frame_id"   type="string" value="base_link" />

  <param name="viz_path"         type="bool" value="true" />
  
  <node name="imu_gnss_fusion_node" pkg="imu_gnss_fusion" type="imu_gnss_fusion_dual_node" output="screen"/>

  <node pkg="rviz" type="rviz" name="rviz" output="screen" 
    args="-d $(find imu_gnss_fusion)/rviz/default.rviz" required="true">
  </node>
</launch>